src/util/webextensionRPC.test.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-env jest */
+
+import { remoteFunction } from './webextensionRPC'
+
+describe('remoteFunction', () => {
+    test('should return a remotely callable function', () => {
+        const remoteFunc = remoteFunction('remoteFunc', {tabId: 1})
+        expect(remoteFunc.name).toBe('remoteFunc_RPC')
+    })
+
+    test('should throw an error when remoteFunc is called', async () => {
+        expect.assertions(1)
+        const remoteFunc = remoteFunction('remoteFunc', {tabId: 1})
+        try {
+            await remoteFunc()
+        } catch (err) {
+            expect(err.toString()).toBe(`Error: Got no response when trying to call 'remoteFunc'. Did you enable RPC in the tab's content script?`)
+        }
+    })
+
+    test('should call the browser.tabs function when tabId is valid', async () => {
+        expect.assertions(2)
+        browser.tabs = {
+            sendMessage: jest.fn(),
+        }
+        browser.runtime = {
+            sendMessage: jest.fn(),
+        }
```

Comment:
Could these two mocks be set in a `beforeEach`? They seem required generally. In some tests you would then just have to set their return values (or perhaps throw an error when called).